internal/db/sqltest/tests/session/session_state_test.sql

```diff
@@ -0,0 +1,40 @@
+begin;
+    select plan(9);
+    select wtt_load('widgets', 'iam', 'kms', 'auth', 'hosts', 'targets', 'credentials','sessions');
+
+    -- Ensure session state table is populated
+    select is(count(*), 1::bigint) from session_state where session_id = 's1____warren';
+    select is(count(*), 1::bigint) from session_state where session_id = 's1____warren' and state='pending';
+    select is(count(*), 1::bigint) from session_state where session_id = 's1____warren' and prior_state='pending';
+
+    -- Disabling trigger; insert_session_state() uses now() and causes pkey violation otherwise
+    ALTER TABLE session_state DISABLE TRIGGER insert_session_state;
```

Comment:
Just leaving a note here that we talked offline about an approach to avoid this disable trigger.